src/com/amazon/corretto/crypto/provider/AmazonCorrettoCryptoProvider.java

```diff
@@ -84,6 +84,10 @@ private void buildServiceMap() {
                     .setSelfTest(AesCtrDrbg.SPI.SELF_TEST);
         }
 
+        addService("SecureRandom", "LibCryptoRng", "LibCryptoRng$SPI",
+                singletonMap("ThreadSafe", "true"), "DEFAULT")
+                .setSelfTest(LibCryptoRng.SPI.SELF_TEST);
+
```

Comment:
FYI, but putting this at the end it has "NIST800-90A/AES-CTR-256" be the default algorithm for `SecureRandom`. However, since we're planning on removing that anyway later, I think it's fine.

Reply:
Fixed in latest commit